app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
I had the impression that there additional information are encoded. Or did I miss the place where a notification payload (HTTP request body) is defined?


--Viele GrüßeHolger Reif

-------- Ursprüngliche Nachricht --------
Von: Thomas Boerger <notifications@github.com> 
Datum: 13.04.2016  09:16  (GMT+01:00) 
An: SUSE/Portus <Portus@noreply.github.com> 
Cc: Holger Reif <holger-git@reif.net> 
Betreff: Re: [SUSE/Portus] [WIP] Webhook integration (#814) 

In app/models/webhook_header.rb:
> @@ -0,0 +1,22 @@
> +# == Schema Information
> +#
> +# Table name: webhook_headers
> +#
> +#  id         :integer          not null, primary key
> +#  webhook_id :integer
> +#  name       :string(255)
> +#  value      :string(255)

I don't think that anybody would send additional header values with a length >255 characters.

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub